src/components/mainMenu.js

```diff
@@ -0,0 +1,21 @@
+import mainMenuButton from "./mainMenuButton";
+
+const mainMenu = (callback) => {
```

Comment:
Tutaj powinnaś w nazwie przekazać informację, do czego służy ten callback. Chodzi między innymi o to, że jak ktoś będzie chciał użyć Twojego komponentu, to powinien wiedzieć, do czego ten callback służy. Zwykle metodologia tworzenia jego nazwy jest taka: ```on[nazwaAkcji]``` Możesz przykładowo nazwać go tak:

```suggestion
const MainMenu = (onButtonClick) => {
```